mmpose/core/optimizer/builder.py

```diff
@@ -0,0 +1,56 @@
+from mmcv.runner import build_optimizer
+
+
+def build_optimizers(model, cfgs):
+    """Build multiple optimizers from configs.
```

Comment:
is this needed because of "multiple optimizers"?

Reply:
Yes. The training of HMR model involves adversarial training and this needs individual optimizers for generator and discrinimator.